private/data/Shops.json

```diff
@@ -3211,5 +3211,11 @@
     "isCommercial": true,
     "isShippingDefault": true,
     "isBillingDefault": true
-  }]
+  }],
+  "defaultParcelSize": {
```

Comment:
I wonder if in addition to doing this if we shouldn't create auto-values on Shop? We sort of talked about this but now I am seeing want to also deal with UOM that aren't lbs.

Reply:
I didn't understand what you meant here.